system/screen.h

```diff
@@ -46,9 +46,21 @@ typedef enum {
 } screen_colour_t;
 
 /**
- * BIOS/UEFI(GOP) agnostic framebuffer copy
+ * Colour Palette definition
  */
+typedef struct {
+    screen_colour_t background;
+    screen_colour_t foreground;
```

Comment:
Suggestion: we can just use "f_" and "b_" prefixes if we prefer shorter names...